cassiopeia/spatial/spatial_imputation.py

```diff
@@ -0,0 +1,125 @@
+"""
+Functionality for spatial imputation.
+"""
+
+from typing import Optional
+
+import anndata
+import networkx as nx
+import numpy as np
+import pandas as pd
+import tqdm
+
+from cassiopeia.spatial import spatial_utilities
+
+
+def impute_spatial_data(
```

Comment:
Is there a better name for this function? You are using spatial coordinates to impute lineage data not imputing spatial data. I'm fine we the name if you can't think of anything better.

Reply:
Agreed - renamed to `impute_alleles_from_spatial_data`.